frontend/src/metabase/routes.jsx

```diff
@@ -215,8 +215,6 @@ export const getRoutes = store => (
         <Route path="collection/:slug" component={CollectionLanding}>
           <ModalRoute path="move" modal={MoveCollectionModal} />
           <ModalRoute path="archive" modal={ArchiveCollectionModal} />
-          <ModalRoute path="new_collection" modal={CollectionCreate} />
-          <ModalRoute path="new_dashboard" modal={CreateDashboardModal} />
```

Comment:
Similarly to removing `newCollection` in:
https://github.com/metabase/metabase/blob/5e16a473a187363454a06e834672421b8bb2425d/frontend/src/metabase/lib/urls/collections.ts#L12-L13

I assume you should also remove `newDashboard` in:
https://github.com/metabase/metabase/blob/5e16a473a187363454a06e834672421b8bb2425d/frontend/src/metabase/lib/urls/dashboards.ts#L10-L11

Since it's also not imported anywhere.

Reply:
Thank you! Removed 🗑️